Login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Trainwise - Login page</title>
+
+</head>
+
+<body>
+
+    <h1>Trainwise</h1>
+
+    <h2>Login to your account</h2>
+    <form>
+        <input type="text" title="username" placeholder="username" />
```

Comment:
inputs need a `name` attribute to pass parameters to the backend

Reply:
I think I need to find some guide on this (how the interaction between the frontend and backend works)